lib/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -54,7 +55,8 @@ public class UserIdentityService {
     private static final String DRIVING_PERMIT_PROPERTY_NAME = "drivingPermit";
     private static final List<String> ADDRESS_CRI_TYPES =
             List.of(ADDRESS_PROPERTY_NAME, "stubAddress");
-    private static final List<String> PASSPORT_CRI_TYPES = List.of("ukPassport", "stubUkPassport");
+    private static final List<String> PASSPORT_CRI_TYPES =
```

Comment:
At some point it might be worth turning this around and having it in configuration. For example, against each issuer URI we could have a list of properties we can get from it, eg "credentialSubject.passport", "evidence", "credentialSubject.drivingPermit".

There's something of a smell from this "type" system ending up with these things getting hard-coded here. It's not urgent to fix, but it's only going to get more messy until we do :-)

Reply:
Yeah, Danny and I talked about this needing an overhaul at some stage. I've been thinking about how much Core should just introspect VCs and go with what is there vs validate what we expect from a particular CRI . Your suggestion sounds like a good balance.

Reply:
This update is intentionally the minimum to get it working.